worker/src/RTC/RtpRetransmissionBuffer.cpp

```diff
@@ -392,14 +380,24 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		return this->Get(this->startSeq);
+		if (this->buffer.empty())
+		{
+			return nullptr;
+		}
+
+		return this->buffer.front();
 	}
 
 	RtpRetransmissionBuffer::Item* RtpRetransmissionBuffer::GetNewest() const
 	{
 		MS_TRACE();
 
-		return this->Get(this->startSeq + this->buffer.size() - 1);
+		if (this->buffer.empty())
+		{
+			return nullptr;
+		}
+
+		return this->buffer.back();
```

Comment:
Oldest seq in the buffer is just the last element by design, period. Let's do it easy.